kafka/producer/buffer.py

```diff
@@ -114,6 +114,7 @@ def queued(self):
         with self._lock:
             return len(self._waiters)
 
+# TODO should this be removed or finished?
```

Comment:
This is fine to remove. Memory allocation is not a bottleneck as far as I can tell.